src/main/resources/application.properties

```diff
@@ -14,4 +14,16 @@ spring.docker.compose.profiles.active=${COMPOSE_PROFILE}
 spring.docker.compose.lifecycle-management=${COMPOSE_LIFECYCLE}
 
 # Jackson configuration
-spring.jackson.property-naming-strategy=SNAKE_CASE
\ No newline at end of file
+spring.jackson.property-naming-strategy=SNAKE_CASE
+
+# Mail configuration
+spring.mail.host=sandbox.smtp.mailtrap.io
+spring.mail.port=2525
+spring.mail.username=c9c80f4256e023
+spring.mail.password=4bf3255b5aaa02
+spring.mail.protocol=smtp
+spring.mail.test-connection=true
+spring.mail.properties.mail.smtp.auth=true
+spring.mail.properties.mail.starttls.enable=true
```

Comment:
I think here we should use env variables here.

- We want to be able to config other service on production in order to send real emails.
- To hide credential from public source code.

Reply:
Okay, I fixed it